src/topology_hwloc.c

```diff
@@ -297,12 +298,18 @@ hwloc_init_cpuInfo(cpu_set_t cpuSet)
     }
     snprintf(cpuid_info.architecture, 19, "armv7");
 #endif
+*/
 #if defined(__ARM_ARCH_7A__) || defined(__ARM_ARCH_8A)
-    uint32_t part = 0;
+    /* uint32_t part = 0; */
```

Comment:
This value is never used.

Reply:
Please remove the line, not comment it out.

Reply:
Okay